doc/classes/FileAccess.xml

```diff
@@ -37,11 +41,11 @@
 		[codeblocks]
 		[gdscript]
 		var file = FileAccess.open("res://something") # File is opened and locked for use.
-		file = null # File is closed.
+		file.close()
```

Comment:
This change is OK, the rest are unnecessary. `close` is optional and can be used if you want to immediately close a file before `File` object exited the current scope, so there's no reason to call it before `return` / at the end of the function.